packages/sdk/src/server/__tests__/server.test.ts

```diff
@@ -564,30 +482,71 @@ describe("ServerClient", () => {
   });
 
   describe("invokeWorkflow", () => {
-    it("should invoke a workflow with provided URL and body", async () => {
-      // Create a mock oauthClient
-      const getTokenMock = jest.fn().mockResolvedValue({
-        token: {
-          access_token: "mocked-oauth-token",
+    it("should invoke a workflow with provided URL and body, with no auth type", async () => {
+      fetchMock.mockResponseOnce(
+        JSON.stringify({
+          result: "workflow-response",
+        }),
+        {
+          headers: {
+            "Content-Type": "application/json",
+          },
+        },
+      );
+
+      const result = await client.invokeWorkflow("https://example.com/workflow", {
+        body: {
+          foo: "bar",
         },
-        expired: jest.fn().mockReturnValue(false),
       });
 
-      const oauthClientMock = {
-        getToken: getTokenMock,
-      } as unknown as ClientCredentials;
+      expect(result).toEqual({
+        result: "workflow-response",
+      });
+      expect(fetchMock).toHaveBeenCalledWith(
+        "https://example.com/workflow",
+        expect.objectContaining({
+          method: "POST",
+          body: JSON.stringify({
+            foo: "bar",
+          }),
+          headers: expect.objectContaining({
+            "Content-Type": "application/json",
+            "X-PD-Environment": "production",
+          }),
+        }),
+      );
+    });
 
-      // Inject the mock oauthClient into the ServerClient instance
-      client = new ServerClient(
+    it("should invoke a workflow with OAuth auth type", async () => {
+      fetchMock.mockResponseOnce(
+        JSON.stringify({
+          result: "workflow-response",
+        }),
         {
-          publicKey: "test-public-key",
-          secretKey: "test-secret-key",
-          oauthClientId: "test-client-id",
-          oauthClientSecret: "test-client-secret",
+          headers: {
+            "Content-Type": "application/json",
+          },
         },
-        oauthClientMock,
       );
 
+      const result = await client.invokeWorkflow("https://example.com/workflow", {}, HTTPAuthType.OAuth);
+
+      expect(result).toEqual({
+        result: "workflow-response",
+      });
+
+      expect(fetchMock).toHaveBeenCalledWith(
+        "https://example.com/workflow",
+        expect.objectContaining({
+          headers: expect.objectContaining({
+            "Authorization": "Bearer mocked-oauth-token",
+          }),
+        }),
+      );
+    });
+
+    it("should invoke a workflow with static bearer auth type", async () => {
```

Comment:
New test for static bearer token (the expectation is that the customer passes this in themselves in the `Authorization` header, as we'll document